test/net/sourceforge/kolmafia/FamiliarDataTest.java

```diff
@@ -38,6 +41,28 @@ public void canTellIfFamiliarIsNotTrainable() {
     assertFalse(fam.trainable());
   }
 
+  @Test
+  void familiarReportsModifiedWeightIncludingFidoxene() {
+    var cleanups = addEffect("Fidoxene");
+
+    try (cleanups) {
+      var familiar = FamiliarData.registerFamiliar(FamiliarPool.ALIEN, 0);
+
+      assertThat(familiar.getModifiedWeight(), equalTo(20));
+    }
+  }
+
+  @Test
+  void familiarReportsModifiedWeightCorrectlyDespiteFidoxene() {
+    var cleanups = addEffect("Fidoxene");
+
+    try (cleanups) {
+      var familiar = FamiliarData.registerFamiliar(FamiliarPool.ALIEN, 4000);
```

Comment:
I don't know what base weight FamiliarData.registerFamiliar(FamiliarPool.ALIEN, 4000) gives but if it is less than 20 this test seems redundant and if it is greater than 20 shouldn't the modified weight also be greater than 20?

Reply:
This is meant to be 400, again sorry. The 400 there is experience not weight, and thus 20 lbs

Reply:
Ultimately this is the same as 400 because experience above that number doesn't count towards weight.